step-release-vis/src/app/components/form/form.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]="dataForm" (ngSubmit)="onSubmit(dataForm.value)">
```

Comment:
form as a filename is not really helpful, can we name this so, that the reader knows from the name what the file is used for?

Reply:
done